src/containers/SharedFolderPage/components/FolderResource.tsx

```diff
@@ -112,8 +114,17 @@ const FolderResource = ({
     () =>
       isLearningPathOrCase
         ? resource.path
+        : preview
+        ? `/minndla/folders/preview/${rootFolderId}/${parentId}/${resource.id}`
```

Comment:
Skal det være parentId her?

Reply:
Ja, til vi ev. gjør en endring på strukturering av lenke. Hvis vi hadde droppa det leddet, ville resourceId havna på samme ledd som subfolderId, og react router har ingen måte for dynamiske ledd å skille mellom om det er den ene eller den andre. Denne er derfor, også under preview, beholdt helt lik som under /folder i URL på linja under.